oneadapter/src/main/java/com/idanatz/oneadapter/OneAdapter.kt

```diff
@@ -79,6 +79,14 @@ class OneAdapter(recyclerView: RecyclerView) {
         }
     }
 
+    fun getItemViewType(position: Int): Int {
```

Comment:
change to Class < Diffable > for the reason written in
ViewHolderCreatorsStore.kt

Reply:
and place these functions at the end of the file